tests/components/__snapshots__/system_notice.test.jsx.snap

```diff
@@ -23,6 +23,18 @@ exports[`components/SystemNotice should match snapshot for admin, admin notice 1
   >
     some body
   </div>
+  <div
+    className="system-notice__info"
+  >
+    <i
+      className="fa fa-eye"
+    />
+    <FormattedMessage
+      defaultMessage="Only visible to System Admins"
+      id="system_notice.adminVisible"
+      values={Object {}}
```

Comment:
Why the empty object?

Reply:
I'm not passing anything to it directly, it's likely that the react-intl library we're is using that as a default, or jest/enzyme is filling it in.